COVID19Tracking/website
# feat: Tweak UX/UI of API, data sections

## To do pre-merge

- [x] Remove images of supporters from about us page content

## Changes

- Change volunteer CTA in mobile to full-width
- Make "more tweets" link on state page open in new window and add "pop out" icon
- Add a large header above gallery that says "A gallery of our charts"
- Move the first lede paragraph of our charts to above the top chart.
- Rewrite API heading content to bullets
- Embed the headway latest changes
- Remove first line of the download page that talks about our public spreadsheet.
- Data definitions page: Add blue treatment to jump to links.
- Rename contact us -volunteering to "Volunteer"
- Put the logos of partners into a grid
- About page: All external links should open a new window
- Add about us blurb to footer
- Change "Totals" to "Total" in chart toggle.
- Change "Full range" in chart toggle to "Historical"
- Remove "Calculated" links in charts
- Move legend for 7-day average and make more explicit
- Larger tap targets for sub navigation& main navigation
- Make footer tap targets bigger
